roles/rsyslog/tasks/outputs/files.yml

```diff
@@ -1,8 +1,50 @@
 ---
+- name: Create __rsyslog_conf_files_output_rules 0
+  set_fact:
+    __rsyslog_conf_files_output_rules:
+      [ { "name": "output-files-default", "type": "output",
+          "sections": [ { "options": "{{ lookup('template', 'output_files_default.j2') }}" } ] } ]
+  loop: '{{ rsyslog_output_files }}'
+  when: item.path is not defined
+
+- name: Create __rsyslog_conf_files_output_rules 1
+  set_fact:
+    __rsyslog_conf_files_output_rules:
+      '{{ __rsyslog_conf_files_output_rules | d([]) }} +
+       [ { "name": "output-files-exc-{{ index }}", "type": "output",
```

Comment:
this is the file name, right? I think it is better to use `item.name` instead of index to construct file names, because the index will change when someone adds another item in the list and the role will report needless changes, while the name is stable.

Reply:
Ok, let me change it.